src/view.tsx

```diff
@@ -0,0 +1,50 @@
+import { VDomRenderer } from '@jupyterlab/apputils';
+/*import { ReactWidget } from '@jupyterlab/apputils';*/
+import { ThemeEditorModel } from './model';
+import React from 'react';
+import Form from '@rjsf/core';
+
+interface IProps {
+  formData: any;
+  schema: any;
+  uiSchema: any;
+  setformData: (value: any) => void;
+}
+
+function FormComponent(props: IProps) {
+  return (
+    <Form
+      schema={props.schema}
+      formData={props.formData}
+      uiSchema={props.uiSchema}
+      onChange={event => {
+        props.setformData(event.formData);
+      }}
+    />
+  );
+}
+
+export class ThemeEditorView extends VDomRenderer<ThemeEditorModel> {
+  public formData: any;
+  public schema: any;
+  public uiSchema: any;
```

Comment:
You don't need those as you can directly use the model attributes in the `render` and it will avoid the error (see below)

```suggestion
```

Reply:
So you mean removing the constructor ? And doing something like this for the `render`method ?
```
render() {
    return (
      <FormComponent
        schema={this.model.schema}
        formData={this.model.formData}
        uiSchema={this.model.uiSchema}
        setformData={(value: any) => {
          formData = value;
          this.update();
        }}
      />
    );
  }
}

 ```
But then `formData` is said to be not known.